CHANGELOG.md

```diff
@@ -3,6 +3,56 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## v5.12.3
+
+_Jun 23, 2022_
+
+We'd like to offer a big thanks to the 7 contributors who made this release possible. Here are some highlights ✨:
+
+- 🌍 Add Swedish (sv-SE) locale on the data grid (#5210) @mrxdst
```

Comment:
If we keep the highlight together we should start to say which product has the new locale

Reply:
Here it's not a problem because he did both the pickers and the data grid at once 🙀😄